portal/src/main/webapp/WEB-INF/jsp/dashboard/dashboard.jsp

```diff
@@ -260,9 +262,12 @@
     	window.cbioURL = window.location.origin + window.location.pathname.substring(0, window.location.pathname.indexOf("/",2)) + '/';
     	window.cbioResourceURL = 'js/src/dashboard/resources/';
 		window.iviz.datamanager = new DataManagerForIviz.init(window.cbioURL, studyCasesMap);
-    
-    
-    	$.when(window.cbioportal_client.getStudies({ study_ids: cohortIdsList}), window.iviz.datamanager.getGeneticProfiles())
+		
+        if(sessionServiceUrl) {
+            vcSession.URL = 'api-legacy/proxy/virtual-cohort';
```

Comment:
better to declare in header.jsp since it would be used in populating virtual cohorts in jstree